css/autotheme.css

```diff
@@ -0,0 +1,68 @@
+.autotheme {
```

Comment:
This class is used on the elements that look like the "Autotheme" action item ("Autotheme     >")